src/main.lua

```diff
@@ -1127,6 +1193,7 @@ addEventingHandler(
 		local from = utils.formatAddress(msg.From)
 		local target = utils.formatAddress(msg.Tags.Target or msg.Tags.Address)
 		local quantity = tonumber(msg.Tags.Quantity)
+		local instantWithdraw = msg.Tags.Instant == true
```

Comment:
```suggestion
		local instantWithdraw = msg.Tags.Instant && msg.Tags.Instant == "true"
```

Reply:
Double && is not valid in lua.  Fixed this

https://github.com/ar-io/ar-io-network-process/pull/97/commits/acdb40836eb10c0afa4556b80f05226df37ddb84

Reply:
[a0c17c3](https://github.com/ar-io/ar-io-network-process/pull/97/commits/a0c17c3c061c235b3e4565caeb92ee4ab155ea8c)